### PR TITLE
Add Invest Section #1

### PR DESCRIPTION
**Pull request by GuiSanches**

### CODE REVIEW

**Comment by GuiSanches** [comment]
image

File: src/sections/Invest/index.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        <CardWrapper>
+            <CardHeader>
+                <IconWrapper>
+                    {parse(icon)}
+                </IconWrapper>
+                <CardTitle>{title}</CardTitle>
+            </CardHeader>
+            <CardContent>{content}</CardContent>
+        </CardWrapper>
+    )
```
Comment (aviggiano) [inline src/sections/Invest/index.tsx:26]: can you move outside of the `Invest` component? 

I think nested components make it harder to read

**Review by aviggiano — CHANGES_REQUESTED** [review]
LGTM, just a simple change

**Comment by aviggiano** [comment]
Also do mobile too

**Comment by tamisapoznik** [comment]
Needs more space at the top and the bottom of the section

File: src/sections/Invest/index.tsx (hunk 1 above)
Comment (GuiSanches) [reply src/sections/Invest/index.tsx]: Sure, thx for the feedback

**Pull request merged**